groups/rds/variables.tf

```diff
@@ -4,6 +4,7 @@ variable "account_name" {
 }
 
 variable "environment" {
+  default     = "devops1"
```

Comment:
We wouldn't specify a default for `environment`. Let this variable be picked up from the profile.